sktime/transformations/series/compose.py

```diff
@@ -96,3 +97,204 @@ def inverse_transform(self, Z, X=None):
         if not self.passthrough:
             z = self.transformer_.inverse_transform(z, X=None)
         return z
+
+
+class ColumnComposition(_SeriesToSeriesTransformer):
+    """
+    Applies a transformer for univariate series
+    columnwise to multivariate series.
+
+    Parameters
+    ----------
+    transformer : Estimator
+        scikit-learn-like or sktime-like transformer to fit and apply to series
+    columns : list of str
+            Names of columns that are supposed to be transformed
+
+     Attributes
+    ----------
+    transformers_ : dict of {str : transformer}
+        Maps columns to transformers
+
+    Example
+    -------
+    >>> from sktime.datasets import base
+    >>> from sktime.transformations.series.detrend import Detrender
+    >>> from sktime.transformations.series.compose import ColumnComposition
+
+    >>> y, X = base.load_longley()
+    >>> transformer = ColumnComposition(Detrender())
+    >>> y_hat = transformer.fit_transform(X)
```

Comment:
```suggestion
    >>> yt = transformer.fit_transform(y, X)
```
y_hat is typically used for estimates/forecasts